src/FixedPairList.cpp

```diff
@@ -274,6 +282,7 @@ namespace espressopp {
       .def("add", pyAdd)
       .def("size", &FixedPairList::size)
       .def("getBonds",  &FixedPairList::getBonds)
+      .def("remove",  &FixedPairList::remove)
```

Comment:
as I said: I am currently using remove, I personally don't see any application of connect disconnect YET! but if it is only to preserve structure could be added. I would say in the future connect disconnect could be added but with a functionality and thus within other PRs...